index.bs

```diff
@@ -1747,6 +1747,9 @@ a numbered step. If outdented, it (today) is rendered either as a bullet in the
 
                         :   "direct"
                         ::  Convey the [=authenticator=]'s [=AAGUID=] and [=attestation statement=], unaltered, to the [=[RP]=].
+
+                        :   "enterprise"
+                        ::  If user agent or authenticator configuration permits it for the requested [=RP ID=], signal to the authenticator that enterprise attestation is permitted and convey the resulting [=AAGUID=] and [=attestation statement=], unaltered, to the [=[RP]=]. If not permitted, act the same as if the value were "direct".
```

Comment:
hm, the imperative "signal to the authenticator" belongs down in the explanation for the `AttestationConveyancePreference` enum because where it is now up in the `[[create]]()` alg, the authnr has _already been invoked_ and we're just  working with the data it has returned, we can't signal it there, it's already been done.

I suggest here we just say the "direct" and "enterprise" do that same thing (see above).

```suggestion
```

Reply:
Good point, thanks. I've (maybe) fixed this change to signal at the correct time.